src/ios/CDVCapture.m

```diff
@@ -292,6 +295,65 @@ - (CDVPluginResult*)processVideo:(NSString*)moviePath forCallbackId:(NSString*)c
     return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:fileArray];
 }
 
+- (void)showAlertIfAccessProhibeted
```

Comment:
Can we change the function name to `showAlertIfAccessProhibited`, just for spelling, please? :)